server/public/styles.css

```diff
@@ -0,0 +1,32 @@
+body {
+  background-color: rgb(240, 240, 240);
+}
+
+.title {
+  background-color: #343434;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+
+}
+
+.titleText {
+  margin: auto;
+  padding: 10px;
+  position: relative;
+  left: 10%;
+  font-size: 14px;
```

Comment:
Bootstrap has lots of spacing helpers. It might even be the best thing about Bootstrap 👍 

https://getbootstrap.com/docs/4.4/utilities/spacing/

and positioning (if you really, really need it)

https://getbootstrap.com/docs/4.4/utilities/position/